appveyor.yml

```diff
@@ -0,0 +1,54 @@
+build: false
+shallow_clone: true
+platform: x86
+clone_folder: c:\projects\phpspec
+
+environment:
+    matrix:
+        - PHP_DOWNLOAD_FILE: php-5.6.14-nts-Win32-VC11-x86.zip
+
+matrix:
+    allow_failures:
+        - PHP_DOWNLOAD_FILE: php-5.6.14-nts-Win32-VC11-x86.zip
+
+skip_commits:
+    message: /\[ci skip\]/
+
+cache:
+    - c:\php -> appveyor.yml
+    - '%LOCALAPPDATA%\Composer'
+    - .phpunit -> phpunit
```

Comment:
this line is useless (we don't have a ``.phpunit`` folder here. Symfony has it because of its special install of PHPUnit due to the Yaml usage in PHPUnit)